tools/update-copyright-notices.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eu
+
+YEAR=$(date +%Y)
+echo "Updating global copyright strings..."
+sed -re 's/(Copyright.*2[0-9]{3}-)[0-9]{4}/\1'$YEAR'/g' -i src/res/translation/*.ts src/util.cpp src/aboutdlgbase.ui
+
+echo "Updating copyright comment headers..."
+for file in $(find -regex '.*\.\(cpp\|h\|mm\)' -not -regex '\./\(\.git\|libs/\|moc_\|ui_\).*'); do
```

Comment:
I just tried this out and found it had the same issue as the recent clang_format one: it found all the files in build directories too.
```suggestion
for file in $(find android ios linux mac src windows -regex '.*\.\(cpp\|h\|mm\)' -not -regex '\./\(\.git\|libs/\|moc_\|ui_\).*'); do
```

Reply:
Thansk, updated.